cardano-node/Cardano/Node/CLI.hs

```diff
@@ -113,7 +113,7 @@ fromProtocol RealPBFT =
     case Consensus.runProtocol p of
       Dict -> return $ SomeProtocol p
   where
-    p = ProtocolRealPBFT defaultDemoPBftParams genesisConfig
+    p = ProtocolRealPBFT defaultDemoPBftParams genesisConfig Nothing
```

Comment:
I believe this `Nothing`(the [`PbftLeaderCredentials`](https://github.com/input-output-hk/ouroboros-network/blob/4e5b99098cc4e4bbf400a9a8e98c42dba6ba574e/ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs#L58)) will break the demo for `--real-pbft`.

A proper solution for this is coming in #60: https://github.com/input-output-hk/cardano-node/pull/60/files?file-filters%5B%5D=.hs#diff-ba9f0f06843a3a67a21c656d60e82bc8R153

So I'm not convinced this PR makes sense. We have already broken the demo too many times by accident, I don't think we should start doing it knowingly :slightly_smiling_face:

I would suggest either waiting for #60 to be merged before updating the dependencies *or* lobbying for #60 to incorporate the dependency bumps from this PR.

Reply:
Yes, let's not break the demo. I can wait for #60 first.

Reply:
#60 is now merged.